src/tools/utils.rs

```diff
@@ -138,7 +138,9 @@ where
 {
     let headers = match header.version() {
         HEADER_VERSION => {
-            let index = BPTreeFileIndex::<K>::from_file(FileName::from_path(path)?, None).await?;
+            let index =
+                BPTreeFileIndex::<K>::from_file(FileName::from_path(path)?, IODriver::default())
```

Comment:
Here the sync `IoDriver` should be created (if `async-io-rio` enabled, then `Rio` should be `None` inside)

Reply:
`IoDriver::new_sync()`